File: src/Frontend/projects/core/src/lib/models/dto/pim/attribute-group.ts
import { Attribute } from './attribute';
export class AttributeGroup {
    public id: number;
    public name: string;
    public attributes?: Attribute[] = [];

    static fromJSON(obj: any) {
        if (!obj) {
            return null;
        }
        return Object.assign(
            new AttributeGroup(),
            obj,
        );
    }
}

File: src/Backend/Common/Models/Pim/AttributeGroup.cs
using System;
using System.Collections.Generic;

namespace Company.Common.Models.Pim
{
    public class AttributeGroup
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public DateTime CreateTime { get; set; }

        public DateTime? DeleteTime { get; set; }

        public int? CreatorId { get; set; }

        public int? DeleterId { get; set; }

        public ICollection<Attribute> Attributes { get; set; } = new List<Attribute>();

        public AttributeGroup()
        {

        }
    }
}
